annotation-processor/src/main/java/com/hubspot/annotation/processor/ModernizerAnnotationProcessor.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2014-2018 Andrew Gaul <andrew@gaul.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hubspot.annotation.processor;
+
+import java.io.IOException;
+import java.io.Writer;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+
+import javax.annotation.processing.AbstractProcessor;
+import javax.annotation.processing.Processor;
+import javax.annotation.processing.RoundEnvironment;
+import javax.annotation.processing.SupportedAnnotationTypes;
+import javax.annotation.processing.SupportedSourceVersion;
+import javax.lang.model.SourceVersion;
+import javax.lang.model.element.Element;
+import javax.lang.model.element.Name;
+import javax.lang.model.element.TypeElement;
+import javax.tools.Diagnostic.Kind;
+import javax.tools.FileObject;
+import javax.tools.StandardLocation;
+
+import com.google.auto.service.AutoService;
+
+@SupportedAnnotationTypes("java.lang.SuppressWarnings")
+@SupportedSourceVersion(SourceVersion.RELEASE_8)
+@AutoService(Processor.class)
+public class ModernizerAnnotationProcessor extends AbstractProcessor {
+    @Override
+    public final boolean process(
+        Set<? extends TypeElement> annotations, RoundEnvironment roundEnv) {
+        List<String> annotatedClasses;
+        for (TypeElement annotation : annotations) {
+            annotatedClasses = getAnnotatedClasses(roundEnv, annotation);
+            makeAnnotatedClassesFile(annotatedClasses);
+        }
+        return true;
+    }
+
+    private List<String> getAnnotatedClasses(
+        RoundEnvironment roundEnv, TypeElement annotation) {
+        List<String> annotatedClasses = new ArrayList<>();
+        for (Element element : roundEnv.getElementsAnnotatedWith(annotation)) {
+            List<String> warnings =
```

Comment:
Is there a reason for a list here?

Reply:
`element.getAnnotation(SuppressWarnings.class).value()` returns `String[]`. I converted it to a list so that I can use `contains()`. Not sure if there's an alternative and better way to do this.

Reply:
Ah got it. I had thought it was a single string.